### PR TITLE
feat(generator):add automatic setup to fullfil #45

### PR DESCRIPTION
**Pull request by agustinf**
This PR adds a Rails Generator that automatically modifies ActiveAdmin JS and CSS files to successfully add ActiveadminAddons.

### CODE REVIEW

File: lib/generators/activeadmin_addons/install/install_generator.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+module ActiveadminAddons
+  module Generators
+    class InstallGenerator < Rails::Generators::Base
+
```
Comment (monkeyci) [inline lib/generators/activeadmin_addons/install/install_generator.rb]: Extra empty line detected at class body beginning.

File: lib/generators/activeadmin_addons/install/install_generator.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+      def add_javascripts
+        inject_into_file 'app/assets/javascripts/active_admin.js.coffee', "#= require activeadmin_addons/all\n", after:"#= require active_admin/base\n"
+      end
+
+      def add_stylesheets
+        prepend_file 'app/assets/stylesheets/active_admin.css.scss', "//= require activeadmin_addons/all\n"
```
Comment (monkeyci) [inline lib/generators/activeadmin_addons/install/install_generator.rb]: Line is too long. [107/100]

File: lib/generators/activeadmin_addons/install/install_generator.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+module ActiveadminAddons
+  module Generators
+    class InstallGenerator < Rails::Generators::Base
+
+      def add_javascripts
+        inject_into_file 'app/assets/javascripts/active_admin.js.coffee', "#= require activeadmin_addons/all\n", after:"#= require active_admin/base\n"
```
Comment (monkeyci) [inline lib/generators/activeadmin_addons/install/install_generator.rb]: Line is too long. [151/100]<br>Space missing after colon.

File: lib/generators/activeadmin_addons/install/install_generator.rb [hunk 4]
```diff
@@ -0,0 +1,15 @@
+module ActiveadminAddons
+  module Generators
+    class InstallGenerator < Rails::Generators::Base
+      def add_javascripts
+        file_path = 'app/assets/javascripts/active_admin.js.coffee'
+        inject_into_file file_path, "#= require activeadmin_addons/all\n", after: "#= require active_admin/base\n"
```
Comment (monkeyci) [inline lib/generators/activeadmin_addons/install/install_generator.rb]: Line is too long. [114/100]

**Comment by agustinf** [comment]
listo @ldlsegovia, quedo ansioso de ver la nueva versión.

**Pull request merged**